### PR TITLE
Liftover blacklists of dm3 and ce10; Other small refactors

### PR DESCRIPTION
**Pull request by chuan-wang**

### CODE REVIEW

**Comment by ewels** [comment]
Ok, would be good to discuss whether we want the new atacseq feature. I was originally for this as it's a minor change and adds additional functionality. The later we go, the more I am wondering whether we should abort this though. The new `atacseq` pipeline has matured quickly and now arguably does a better job of analysing ATACseq data than the chipseq pipeline with `--ATACseq`. I think we're also at risk of making this confusing for users by having two pipelines for the same data type.

@chuan-wang - I know that I said to you literally only an hour or two ago to keep this functionality, but I'm now thinking that actually maybe it's best to keep this pipeline as just a chipseq pipeline.

What do you think? @drpatelh / @nf-core/core - any thoughts?

**Comment by drpatelh** [comment]
@ewels @chuan-wang I also think its best to keep these two pipelines separate. They are very similar in the initial bits of the analysis i.e. alignment, filtering and peak calling, however the downstream steps and QC are fundamentally different.

* You generally remove mitochondrial reads for ATACSeq data
* There may be a separate requirement to shift the reads in future releases of the ATACSeq pipeline
* The QC software and assessment will be different for each
* Merging replicate-level alignments may not be necessary for ChIPSeq because footprinting wont be required
* The downstream software will be different with specific parameter sets

The current `--ATACseq` implementation in the chipseq pipeline is a major oversimplification of the steps that are actually required for this type of analysis.

Basically, its a great idea because they are similar enough during the initial steps but the pipeline will be too long and difficult to follow. I think we should keep them separate to keep things simple and to be able to easily add protocol-specific functionality in the future. Having said that the best solution would actually be to make them as modular as possible for the steps they do have in common. Scientists tend to run ChIPSeq and ATACSeq together, and it would be nice to make them as similar as possible in terms of the processes and software used.

BTW Im not saying this because I have written the `atacseq` pipeline! I just never imagined I would put them in the same pipeline anyway for the reasons I outlined above.

Its all @ewels fault.

**Comment by drpatelh** [comment]
@apeltzer Following on from my comments above these two pipelines are obvious candidates for the new Nextflow modularisation feature. Possible challenge for the nf-core Hackathon in Tubingen?

**Comment by ewels** [comment]
✋ I can take the blame here for changing my mind! Sorry @chuan-wang. Do you think you could strip out the ATAC-seq part of this pull-request? Do you have any thoughts?

**Comment by chuan-wang** [comment]
Hi @ewels, the ATAC-seq related contents have been removed.
